docs/design/autoware-architecture/planning/index.md

```diff
@@ -168,6 +168,7 @@ For more details, please refer to the design documents in each package.
   - [_virtual_traffic_light_](https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_velocity_virtual_traffic_light_module/)
   - [_occlusion_spot_](https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_velocity_occlusion_spot_module/)
   - [_run_out_](https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_velocity_run_out_module/)
+  - [_no_drivable_lane_](https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_velocity_no_drivable_lane_module/docs/no-drivable-lane-design.md)
```

Comment:
lets change this link with the github.io one. We should wait until https://github.com/autowarefoundation/autoware.universe/pull/3967 is merged and sync process is done

Reply:
@ahmeddesokyebrahim I am afraid you forgot to update this link

Reply:
Thank you :)